acceptance/tests/facts/windows.rb

```diff
@@ -22,7 +22,10 @@
     os_version  = '2012 R2'
     kernel_version = '6.3'
   elsif agent['platform'] =~ /-10/
-    os_version  = /^10\./
+    os_version  = '10'
```

Comment:
Ah right. I think what I'm saying is that having the major version fact value equal to the full version fact value is a regression on consumer Windows OS versions. In Facter 2.x, these two facts didn't report the same value. Ie, here's the relevant output from facter on Windows 10 Enterprise:
```
C:\Users\moses\development\puppet [master ≡]> bundle exec facter os
{"name"=>"windows", "family"=>"windows", "release"=>{"major"=>"10", "minor"=>"0", "full"=>"10.0.10586"}}
C:\Users\moses\development\puppet [master ≡]> bundle exec facter kernelmajversion
10.0
C:\Users\moses\development\puppet [master ≡]> bundle exec facter kernelversion
10.0.10586
C:\Users\moses\development\puppet [master ≡]> bundle exec facter kernelrelease
10.0.10586
C:\Users\moses\development\puppet [master ≡]> bundle exec facter --version
2.4.6
C:\Users\moses\development\puppet [master ≡]>
```